### ISSUE TITLE
[fix]: Make icons render accessibly and as intended by USWDS

### ISSUE THREAD

**Issue by werdnanoslen** [trussworks/react-uswds#2811]
**ReactUSWDS Version & USWDS Version:**

[Has existed for some time]

**Describe the bug**

Icons in react-uswds do not have critical a11y features as [described here by USWDS](https://designsystem.digital.gov/components/icon/#accessibility-guidance).

**To Reproduce**

1. Go to any individual Icon story in storybook
2. Click on the Accessibility tab
3. See serious issues

**Expected behavior**

Each Icon should have...
* `aria-hidden` if icons are to be used on their own for visual purposes, exposed as a prop. If `false` (not hidden), the following should kick in...
* `title` inside the `<svg>`
* `aria-labelledby` referencing the above `title` by `id`

**Additional context**

These are 508 violations
